erpnext/projects/doctype/project/project.py

```diff
@@ -55,9 +55,18 @@ def copy_from_template(self):
 			if not self.project_type:
 				self.project_type = template.project_type
 
-			# create tasks from template
+			pt_set=set()
 			for task in template.tasks:
+				if task.parent_task:
+					pt_set.add(task.parent_task)
+			pt_list=list(pt_set)
+			task_no={}
+
+			# create Parent Tasks from template tasks list
+			for i in pt_list:
```

Comment:
Avoid using `i` for naming your variables, this could be:

```suggestion
			for task_idx in pt_list:
```